client/components/forms/select-opt-groups.jsx

```diff
@@ -1,32 +1,33 @@
 /**
  * External dependencies
  */
-var React = require( 'react' ),
-	debug = require( 'debug' )( 'calypso:forms:select-opt-groups' );
+import React, { Component } from 'react';
+import { omit } from 'lodash';
+const debug = require( 'debug' )( 'calypso:forms:select-opt-groups' );
```

Comment:
All right! I had this doubt and I asked about it in [an earlier PR](https://github.com/Automattic/wp-calypso/pull/7551#issuecomment-241041618). Gonna fix it in a minute!